include/circt/Dialect/SV/TypeDecl.td

```diff
@@ -222,6 +218,33 @@ def GetModportOp: SVOp<"modport.get", [NoSideEffect]> {
   ];
 }
 
+def AssignInterfaceSignalOp : SVOp<"interface.signal.assign", []> {
+  let summary = "Assign an interfaces signal to some other signal.";
+
+  let arguments = (ins InterfaceType:$iface, FlatSymbolRefAttr:$signalName,
+    AnyType:$rhs);
+
+  let assemblyFormat = [{
+    $iface `->` $signalName `=` $rhs attr-dict `:` type($iface) `,` type($rhs)
+  }];
+}
+
+def ReadInterfaceSignalOp : SVOp<"interface.signal.read", [NoSideEffect]> {
+  let summary = "Access the data in an interface's signal.";
+
+  let arguments = (ins InterfaceType:$iface, FlatSymbolRefAttr:$signalName);
+  let results = (outs AnyType:$signalData);
+
+  let assemblyFormat = [{
+    $iface `->` $signalName attr-dict `:`
```

Comment:
I know it's just the assembly format, but I think I slightly prefer

 ```$iface `.` $signalName```

I'm just suggesting it this way since the same was brought up in the struct PR [here](https://github.com/llvm/circt/pull/307#discussion_r540439350), and it seems like whatever we choose, we should use it here, in the struct PR, and anywhere else we do this sort of field access type operation.

Reply:
I would have preferred that as well, unfortunately [`.` isn't supported by assemblyFormat](https://mlir.llvm.org/docs/OpDefinitions/#literals). I was thinking about switching to (). I also don't like the type syntax at the end.

Reply:
Ah fair enough, this syntax makes sense.

Reply:
What do you think about something like: `%0 = %ifaceInstanceResult ( @handshake_example::@data ) : i32`?

Reply:
```
    %ifaceData = sv.interface.signal.read %iface (@handshake_example::@data) : i32
    %zero32 = constant 0 : i32
    sv.interface.signal.assign %iface(@handshake_example::@data) = %zero32 : i32
```

Would be the test. What say you?

Reply:
I think that looks good.